src/MSBuild/Resources/Strings.resx

```diff
@@ -730,12 +730,30 @@ Copyright (C) Microsoft Corporation. All rights reserved.
                      that the project graph be statically discoverable at
                      evaluation time, but can improve scheduling and reduce
                      memory overhead when building a large set of projects.
+                     (Short form: /isolate)
     </value>
     <comment>
       LOCALIZATION: "MSBuild" should not be localized.
       LOCALIZATION: "/isolateProjects" should not be localized.
       LOCALIZATION: None of the lines should be longer than a standard width console window, eg 80 chars.</comment>
   </data>
+  <data name="HelpMessage_36_GraphBuildSwitch" UESanitized="false" Visibility="Public">
+    <value>  /graphBuild[:True|False]
+                     Causes MSBuild to construct and build a project graph.
+
+                     Constructing a graph involves identifying project
+                     references to form dependencies. Building that graph
+                     involves attempting to build project references prior
+                     to the projects that reference them, differing from
+                     traditional MSBuild scheduling.
```

Comment:
This is good for now. I suspect eventually we'll probably want something more like "this imposes requirements on your project, see aka.ms/msbuild/static for details".

Reply:
At some point prior to v16 release we'll probably want to do a full pass of all the user-facing strings we're adding with these features, potentially with a technical writer or something.